flags/flags.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package flags contains the top level flags for all commands.
+package flags
+
+import "github.com/abcxyz/pkg/cli"
+
+// GlobalFlags represent the shared global flags among all sub-commands. Embed this
+// struct into all sub-commands.
+type GlobalFlags struct {
```

Comment:
My general rule is that there's no such thing as a truly global flag; you'll have a situation where the flag doesn't actually make sense. So it's usually better to name these with a more realistic taxonomy and then embed them.

You did the embedding, but I don't think "global flags" is a good name.

Reply:
How does `AutomationFlags` sound?